test/terramate_provider_versions.tm.hcl

```diff
@@ -0,0 +1,5 @@
+globals {
+  provider                    = "google"
+  minimum_provider_version    = "4.0"
+  provider_version_constraint = "~> ${global.minimum_provider_version}"
+}
```

Comment:
this file is not part of the template repository. please use versions.tm.hcl from template repository.